sklearnex/cluster/dbscan.py

```diff
@@ -48,7 +49,7 @@ def _save_attributes(self):
 
 
 @control_n_jobs(decorated_methods=["fit"])
-class DBSCAN(PatchableEstimator, _sklearn_DBSCAN, BaseDBSCAN):
+class DBSCAN(BaseDBSCAN, _sklearn_DBSCAN):

```

Comment:
This is not obvious why this inheritance scheme is compliant with the previous statement: "The next object in the Estimators MRO after oneDALEstimator should be the equivalent sklearn estimator, if it is BaseEstimator, it is a sklearnex-only estimator."

It would be nice to add a descriptive comment somewhere about what the inheritance scheme for sklearnex estimators should look like in the case when there _is_ the equivalent sklearn estimator and when there _is no_ the equivalent estimator.
From the first glance it is not obvious why oneDALEstimator precedes _sklearn_DBSCAN in this particular inheritance scheme.